### PR TITLE
feature(condo): DOMA-9646 add footnote in tour

### PR DESCRIPTION
**Pull request by tolmachev21**
Before: 
image

After: 
telegram-cloud-photo-size-2-5318798821632893249-y

### CODE REVIEW

File: apps/condo/lang/ru/ru.json
Comment (SavelevMatthew) [inline apps/condo/lang/ru/ru.json:2316]: What’s this? Where is en translation? And why some Avalone in open source translations ?

**Review by SavelevMatthew — CHANGES_REQUESTED** [review]

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [inline apps/condo/pages/tour/guide.tsx:191]: why `ru` only?

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [inline apps/condo/pages/tour/guide.tsx:205]: What is this?

File: apps/condo/next.config.js
Comment (Alllex202) [inline apps/condo/next.config.js:58]: where variable is `GUIDE_MODAL_CARD_EXAMPLES`?

Comment (tolmachev21) [reply apps/condo/next.config.js:58]: Okey, how I should name this variable?

Comment (Alllex202) [reply apps/condo/next.config.js:58]: Where is this variable in `.helm`? I didn't find it

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [inline apps/condo/pages/tour/guide.tsx:201]: text without translate?

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [inline apps/condo/pages/tour/guide.tsx:93]: Please describe the structure of your config or add example. 

Each locale should have its own text/links/pictures.
It shouldn't be that in the English interface I will have a picture with Russian letters

Comment (tolmachev21) [reply apps/condo/pages/tour/guide.tsx:93]: This changes only for ru locale. In en locale nothing should happen

File: apps/condo/next.config.js
Comment (tolmachev21) [reply apps/condo/next.config.js:58]: In `.helm` I am waiting for approval of the changes

Comment (tolmachev21) [reply apps/condo/next.config.js:58]: How I can this name this banner? 
image

File: apps/condo/pages/tour/guide.tsx
Comment (tolmachev21) [reply apps/condo/pages/tour/guide.tsx:201]: @toplenboren Told me that this translation should have been in the env var as well

File: apps/condo/next.config.js
Comment (toplenboren) [reply apps/condo/next.config.js:58]: just idk managementCompanyReview banner

File: apps/condo/pages/tour/guide.tsx
Comment (toplenboren) [reply apps/condo/pages/tour/guide.tsx:191]: Yep, best to make smth like

modal_card_examples_lalala: { en: ... ru: ... }

File: apps/condo/pages/tour/guide.tsx
Comment (toplenboren) [reply apps/condo/pages/tour/guide.tsx:201]: Whole card should be hidden if there is no translate

File: apps/condo/pages/tour/guide.tsx
Comment (toplenboren) [reply apps/condo/pages/tour/guide.tsx:203]: best to use lodash get so your frontend wont crash if var is missing

File: apps/condo/next.config.js
Comment (tolmachev21) [reply apps/condo/next.config.js:58]: `guideManagementCompanyReview` this name be better? @Alllex202

Comment (Alllex202) [reply apps/condo/next.config.js:58]: > In `.helm` I am waiting for approval of the changes

Add these changes to this pr.
You should switch the submodule to your commit with changes

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [reply apps/condo/pages/tour/guide.tsx:93]: > This changes only for ru locale. In en locale nothing should happen

It's better to support any locale.
If tomorrow they tell you we want hints in English or any other language, then for this you only need to update envs.

And we are also open source :)

File: apps/condo/next.config.js
Comment (Alllex202) [reply apps/condo/next.config.js:58]: > `guideManagementCompanyReview` this name be better? @Alllex202

I don't think it's related to company reviews. Look where all the links listed lead.
These are just links to blog posts. Some with company reviews, some without

Comment (Alllex202) [reply apps/condo/next.config.js:58]: > `guideManagementCompanyReview` this name be better? @Alllex202

How about `guideModalCardReviews`?

File: apps/condo/pages/tour/guide.tsx
Comment (SavelevMatthew) [inline apps/condo/pages/tour/guide.tsx:194]: Why do you use `{}` as a fallback? Src should be a string, shouldn't it?

File: apps/condo/pages/tour/guide.tsx
Comment (SavelevMatthew) [inline apps/condo/pages/tour/guide.tsx:195]: where is alt?(

File: apps/condo/public/onboarding/guide/aboutApp/ru/extraIncome/modalLogo.webp
Comment (SavelevMatthew) [inline apps/condo/public/onboarding/guide/aboutApp/ru/extraIncome/modalLogo.webp:1]: what's this?

**Review by SavelevMatthew — APPROVED** [review]

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [inline apps/condo/pages/tour/guide.tsx:204]: Looks like this should be a link text :)
```suggestion
                                                        label: get(guideModalCardReviews, [locale, 'linkText']),
```

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [inline apps/condo/pages/tour/guide.tsx:202]: Maybe it would be better to call the property `postUrl` or `reviewUrl`?

File: apps/condo/pages/tour/guide.tsx
Comment (Alllex202) [reply apps/condo/pages/tour/guide.tsx:204]: Should this translation be added to the envs?
It seems it will always be the same no matter what reviews you add

**Review by Alllex202 — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**
